pyteal/__init__.pyi

```diff
@@ -149,6 +149,7 @@ __all__ = [
     "SetBit",
     "SetByte",
     "Sha256",
+    "Sha3_256",
```

Comment:
@jdtzmn Minor:  After approving I realized I missed user guide consideration.  Can you extend https://pyteal.readthedocs.io/en/stable/crypto.html to include `Sha3_256`?